packages/shylu/shylu_dd/frosch/src/CoarseSpaces/FROSch_CoarseSpace_def.hpp

```diff
@@ -150,10 +150,10 @@ namespace FROSch {
             }
         }
 
-        UnassembledBasesMaps_.resize(0);
-        UnassembledBasesMapsUnique_.resize(0);
-        UnassembledSubspaceBases_.resize(0);
-        Offsets_.resize(0);
+        UnassembledBasesMaps_.clear();
```

Comment:
This doesn't necessarily free anything.  The standard idiom for ensuring that all the elements of a `std::vector` (or `Teuchos::Array`) get freed is to swap the vector with an empty vector.

Reply:
See above.